src/app/client/src/app/modules/learn/components/batch/batch-details/batch-details.component.html

```diff
@@ -1,102 +1,133 @@
-                <div class="item py-10">
-                    <div class="content width-100">
-                        <button
-                            [ngClass]="{'sb-btn-disabled': isUnenrollbtnDisabled,'sb-btn-outline-primary': !isUnenrollbtnDisabled}"
-                            class="ui right floated sb-btn sb-btn-normal ml-8" appTelemetryInteract
-                            [telemetryInteractEdata]="unenrollBatchIntractEdata"
-                            [telemetryInteractObject]="{id:enrolledBatchInfo.identifier,type:'CourseBatch',ver:'1',rollup:{l1:courseId}}"
-                            (click)="unenrollBatch(enrolledBatchInfo)"
-                            [disabled]="isUnenrollbtnDisabled">{{resourceService.frmelmnts.btn.unenroll}}</button>
-                        <span class="header pb-5">{{enrolledBatchInfo?.name}} </span>
-                        <div class="selectedBatchMeta meta">
-                            <span class="date">{{enrolledBatchInfo.startDate | dateFormat}} </span>
-                            <span *ngIf="enrolledBatchInfo.endDate" class="date"> -
-                                {{enrolledBatchInfo?.endDate | dateFormat}}</span>
+<!-- create batch -->
+<sui-accordion class="styled fluid sb-accordion mb-16" *ngIf="courseMentor">
+    <sui-accordion-panel [isOpen]="true">
+        <div title class="sb-accordion-title fsmall d-flex flex-ai-center">
+            {{resourceService?.frmelmnts?.lbl?.batches}}
+            <i class="chevron down icon ml-auto"></i>
+        </div>
+        <div content class="sb-accordion-content sb-bg-color-white pt-0 pb-16">
+            <div class="batch-details">
+                <div class="mb-16" *ngIf="showBatchList">
+                    <div class="batch-details__dropdown">
+                        <sui-select (selectedOptionChange)="getAllBatchDetails()" class="selection mb-16"
+                            [(ngModel)]="batchStatus" [options]="statusOptions" valueField="value" labelField="name"
+                            #batchStatusSelect>
+                            <sui-select-option *ngFor="let option of batchStatusSelect?.filteredOptions"
+                                [value]="option">
+                            </sui-select-option>
+                        </sui-select>
+                        <div *ngFor="let batch of batchList">
+                            <div class="batch-details__edit d-flex flex-ai-center">
+                                <div class="sb-color-primary fmedium font-weight-bold"><i
+                                        class="calendar outline icon"></i> {{batch.startDate | dateFormat}}
+                                    <span *ngIf="batch.endDate"> - {{batch.endDate | dateFormat}} </span></div>
+                                <button class="sb-btn sb-btn-link sb-btn-link-primary sb-btn-normal ml-auto"
+                                    appTelemetryInteract [telemetryInteractEdata]="updateBatchIntractEdata"
```

Comment:
This event also should have the cdata which should include the course details

Reply:
Done